src/routes/todo.routes/index.ts

```diff
@@ -1,16 +1,16 @@
 import { Router } from "express";
-import createToDoItem from "./createToDo";
-import deleteToDoItem from "./deleteToDo";
-import { getToDoItem } from "./getToDo";
-import getToDosItems from "./getToDos";
+import { todoRouter as createToDoItem } from "./createToDo";
```

Comment:
Let's see how we can avoid giving aliases to the imports. They sometimes cause a lot of confusion.

If you're gonna keep the file-per-endpoint approach, i would recommend refactoring like this:

- Each file contains a function. Let's take the `getToDo` endpoint. Then you would have `getToDo.ts` file which contains a function  `function getToDoRoute(router: Router)`
- That function has one parameter - router. That means you can define an endpoint inside of it: `router.get(/, ...))`